rcl_yaml_param_parser/src/impl/parse.h

```diff
@@ -53,7 +53,7 @@ RCL_YAML_PARAM_PARSER_PUBLIC
 RCUTILS_WARN_UNUSED
 rcutils_ret_t parse_key(
   const yaml_event_t event,
-  uint32_t * map_level,
+  yaml_map_lvl_t * map_level,
```

Comment:
This does never need to be `uint32_t`, but `yaml_map_lvl_t`. this allows the compiler to give us the warning, and we do not even do the test for the invalid argument.